utils/netutil/netutil.go

```diff
@@ -17,12 +17,13 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"strings"
 	"time"
 )
 
 // _supportedInterfaces is an ordered list of ip interfaces from which
 // host ip is determined.
-var _supportedInterfaces = []string{"eth0", "ib0"}
+var _supportedInterfaces = []string{"eth0", "ib0", "ethernet", "Ethernet", "vEthernet", "vethernet"}
```

Comment:
Does this also need `vEthernet (Default Switch)` just in case?

Reply:
No, if you look a few lines down I trim the interface name at the first space. I found that the "name" bit varied depending on where is was running so was best to ignore it